### ISSUE TITLE
Monkey test crash on iOS14.0, iOS14.2beta

### ISSUE THREAD

**Issue by taonhannguyen** [zalando/SwiftMonkey#95]
Hello.
While SwiftMonkey seems to work great on iOS13 simulator, they are crashing in iOS14.0 and 14.2beta simulators, using either Xcode 12.0 and 12.2 beta

![Screen Shot 2020-09-30 at 3 48 15 PM](https://user-images.githubusercontent.com/39965249/94732896-ea5bce80-0334-11eb-976c-444b33bba186.png)

![Screen Shot 2020-09-30 at 3 56 13 PM](https://user-images.githubusercontent.com/39965249/94733264-7cfc6d80-0335-11eb-8763-c624a78ed4b6.png)

**Comment by taonhannguyen** [comment]
`        let monkey = Monkey(frame: app.frame)
        monkey.addDefaultUIAutomationActions()
        monkey.addXCTestTapAlertAction(interval: 100, application: app)
        monkey.monkeyAround()`

**Comment by wojciechczerski** [comment]
Hey, could you try replacing the `monkey.addDefaultUIAutomationActions` with `monkey.addDefaultXCTestPublicActions(app: application)`?

**Comment by taonhannguyen** [comment]
Hi there
First and foremost, thank you for your reply. Indeed, this has solved the trick and it works.
That said, I would like to mention for those people using cocoapod that this API change is not available on release 2.1.1.
The function you are describing has been implemented 1 month after release 2.1.1, so to get it I had to replace 

pod 'SwiftMonkey'
by
pod 'SwiftMonkey', :git => 'https://github.com/zalando/SwiftMonkey.git'

Thanks again for the help!
